scripts/build.sh

```diff
@@ -31,5 +31,5 @@ TESTS=$2
 # TODO Add handling for error codes
 mkdir -p build
 cd build
-cmake .. -DCMAKE_BUILD_TYPE=$1 -DTESTS=$2
+cmake .. -DCMAKE_BUILD_TYPE=$1 -DTESTS=$2 -DEVMJIT=On -DLLVM_DIR=/usr/lib/llvm-3.9/lib/cmake/llvm
```

Comment:
This is a Linux-specific path which is wrong on macOS, so we'll need some conditionals in the bash for this.